modules/game/utils.ts

```diff
@@ -7,3 +8,17 @@ export const playerToMinimalPlayer = <T extends MinimalPlayer>(
   rating: player.rating,
   games: player.games,
 });
+
+export const getEliminationFoul = (
+  teams: TeamScoring[],
+  eliminationType: string = ""
+) => {
+  let eliminationFoul = "";
+
+  teams.forEach((team) => {
+    if (team.eliminationFoul === eliminationType) {
+      eliminationFoul = team.eliminationFoul;
+    }
+  });
+  return eliminationFoul;
```

Comment:
Au lieu d'utiliser une let je pense qu'on pourrait faire une for loop avec un return, et si rien n'est return dans la loop on return null